test/bigint64.test.ts

```diff
@@ -1,5 +1,6 @@
 import assert from "assert";
-import { encode, decode } from "../src";
+import { encode, decode, IntMode, ExtensionCodec, DecodeError } from "../src";
+import { getInt64, getUint64 } from "src/utils/int";
```

Comment:
```suggestion
import { getInt64, getUint64 } from "../src/utils/int";
```

I believe the CI browser tests are complaining about this